lib/withdrawal.dart

```diff
@@ -363,6 +367,8 @@ class _WithdrawalFormScreenState extends State<WithdrawalFormScreen> {
                               validator: (value) {
                                 if (value == null || value.isEmpty)
                                   return 'Please enter a value';
+                                value = value.trim();
+                                Clipboard.setData(ClipboardData(text: value));
```

Comment:
why are you copying to the clipboard here?

Reply:
Just realized I forgot to remove it. Was doing it for debugging to see if it was really an empty string or not.